zio-quickstart-kafka/src/main/scala/dev/zio/quickstart/StreamingKafkaApp.scala

```diff
@@ -31,20 +31,25 @@ object StreamingKafkaApp extends ZIOAppDefault {
       ZStream
         .repeatZIO(Clock.currentDateTime)
         .schedule(Schedule.spaced(1.second))
-        .map(time => new ProducerRecord(KAFKA_TOPIC, time.getMinute, s"$time -- Hello, World!"))
+        .map(time =>
+          new ProducerRecord(
+            KAFKA_TOPIC,
+            time.getMinute,
+            s"$time -- Hello, World!"
+          )
+        )
```

Comment:
Usually, we use `{}` for a multi-line argument:
```suggestion
        .map { time =>
          new ProducerRecord(
            KAFKA_TOPIC,
            time.getMinute,
            s"$time -- Hello, World!"
          )
        }
```

Reply:
Thanks, fixed